src/Datadog.Trace.Ci.Shared/CIEnvironmentValues.cs

```diff
@@ -107,19 +158,71 @@ public static void DecorateSpan(Span span)
             }
 
             span.SetTag(CommonTags.CIProvider, Provider);
-            span.SetTag(CommonTags.CIPipelineId, PipelineId);
-            span.SetTag(CommonTags.CIPipelineName, PipelineName);
-            span.SetTag(CommonTags.CIPipelineNumber, PipelineNumber);
-            span.SetTag(CommonTags.CIPipelineUrl, PipelineUrl);
-            span.SetTag(CommonTags.CIJobUrl, JobUrl);
-            span.SetTag(CommonTags.CIWorkspacePath, WorkspacePath);
-
-            span.SetTag(CommonTags.GitRepository, Repository);
-            span.SetTag(CommonTags.GitCommit, Commit);
-            span.SetTag(CommonTags.GitBranch, Branch);
-            span.SetTag(CommonTags.GitTag, Tag);
-
-            span.SetTag(CommonTags.BuildSourceRoot, SourceRoot);
+
+            if (!string.IsNullOrEmpty(PipelineId))
+            {
+                span.SetTag(CommonTags.CIPipelineId, PipelineId);
+            }
```

Comment:
NIT: might be worth introducing a helper method 😛

Reply:
done in 3215416